app/models/concerns/foreman/thread_session.rb

```diff
@@ -73,9 +73,9 @@ def current=(o)
             user = o.login
             type = o.admin? ? 'admin' : 'regular'
             if o.hidden?
-              Rails.logger.debug("Current user set to #{user} (#{type})")
+              Foreman::Logging.logger('taxonomy').debug("Current user set to #{user} (#{type})")
             else
-              Rails.logger.info("Current user set to #{user} (#{type})")
+              Foreman::Logging.logger('taxonomy').info("Current user set to #{user} (#{type})")
```

Comment:
Would it be more useful to log the current user in the permissions logger instead?

Reply:
I am fine however permissions are disabled by default.